httpie/plugins/manager.py

```diff
@@ -69,9 +70,14 @@ def iter_entry_points(self, directory: Optional[Path] = None):
 
     def load_installed_plugins(self, directory: Optional[Path] = None):
         for entry_point in self.iter_entry_points(directory):
-            plugin = entry_point.load()
-            plugin.package_name = get_dist_name(entry_point)
-            self.register(entry_point.load())
+            plugin_name = get_dist_name(entry_point)
+            try:
+                plugin = entry_point.load()
+            except BaseException as exc:
+                warnings.warn(f'While loading "{plugin_name}", an error ocurred: {exc}')
```

Comment:
What if we offered a suggestion to `httpie plugins uninstall {plugin_name}`? 

Actually, does `httpie plugins` supportd upgrades 🤔

Reply:
> What if we offered a suggestion to httpie plugins uninstall {plugin_name}?

Makes sense.

> Actually, does httpie plugins support upgrades thinking

Oh, I think we missed that. Let me create an issue for it. (The only way of doing it is uninstall + re-install at the moment).